.pre-commit-config.yaml

```diff
@@ -24,13 +24,13 @@ repos:
   - id: absolufy-imports
 
 # Find common spelling mistakes in comments and docstrings
-- repo: https://github.com/codespell-project/codespell
-  rev: v2.1.0
-  hooks:
-  - id: codespell
-    args: ['--ignore-regex="\b[A-Z]+\b"'] # Ignore capital case words, e.g. country codes
-    types_or: [python, rst, markdown]
-    files: ^(scripts|doc)/
+# - repo: https://github.com/codespell-project/codespell
+#   rev: v2.1.0
+#   hooks:
+#   - id: codespell
+#     args: ['--ignore-regex="\b[A-Z]+\b"'] # Ignore capital case words, e.g. country codes
+#     types_or: [python, rst, markdown]
+#     files: ^(scripts|doc)/
```

Comment:
Why exclude codespell?

Reply:
It suggested some corrections for URLs which I couldn't ignore.